### DISCUSSION TITLE
Group job operation with partitions

### DISCUSSION THREAD

**Discussion by nikitok** [dagster-io/dagster#25977] [Ideas]
In dagster 1.9 i remark that, with run job manually with partitions, this job look like "group" in run section.
But, with run automatically (by schedule), i saw every job with partition in "run section".
How i could run job by scheduled look like manual running (as group? or subgroup?)?

![telegram-cloud-photo-size-2-5321560777662000109-x](https://github.com/user-attachments/assets/6816da15-7f55-4479-bc15-57f7f89b6ddc)

**Comment by garethbrickman** [comment]
The grouping of backfills is a native and automatic feature of the new Runs page, you can't emulate it for other runs. 

Discussion about the new runs page here: https://github.com/dagster-io/dagster/discussions/24898
